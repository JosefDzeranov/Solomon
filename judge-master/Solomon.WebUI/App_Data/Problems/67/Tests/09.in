1000
2948 5544 640006
1480 6562 640006
1594 6808 640006
4710 1537 640006
6286 361 640006
3367 6354 640006
260 4739 640006
145 489 640006
3153 3094 640006
8183 704 640006
2937 475 640006
2352 7503 640006
1038 442 640006
8402 1178 640006
3099 4454 640006
2819 4231 640006
4601 3701 640006
118 2714 640006
7859 1232 640006
5892 3199 640006
799 2568 640006
437 4167 640006
8887 1102 640006
2038 910 640006
462 7261 640006
148 7369 640006
8233 1721 640006
3153 5339 640006
3946 2340 640006
792 7510 640006
3728 121 640006
2948 6602 640006
3436 5198 640006
1620 3699 640006
1439 6747 640006
3574 2072 640006
3574 2318 640006
677 1364 640006
3574 2236 640006
7250 1187 640006
799 1242 640006
2756 2968 640006
1480 70 640006
2906 530 640006
1031 7202 640006
2463 1841 640006
6767 2324 640006
1480 8070 640006
1358 6139 640006
1358 2054 640006
1620 3129 640006
802 7299 640006
1336 3825 640006
3761 4541 640006
7553 591 640006
4421 4280 640006
3407 3643 640006
4076 4767 640006
802 637 640006
2298 639 640006
3079 1622 640006
2038 6281 640006
1038 4787 640006
634 724 640006
2463 5090 640006
6767 483 640006
1550 3512 640006
3728 4458 640006
51 4326 640006
2989 1087 640006
2463 5939 640006
2832 5939 640006
7497 2224 640006
4771 228 640006
2463 1804 640006
437 1330 640006
8887 204 640006
1620 3628 640006
6286 361 640006
1110 3639 640006
9091 863 640006
2756 6032 640006
118 2819 640006
4508 3534 640006
4701 3618 640006
7497 2457 640006
3389 5498 640006
1358 1222 640006
8771 72 640006
2756 680 640006
677 5837 640006
1480 6839 640006
260 3501 640006
2352 6436 640006
792 2936 640006
2505 3219 640006
1038 7596 640006
4126 354 640006
4267 5338 640006
6286 3313 640006
3728 6257 640006
792 7391 640006
2359 2390 640006
7497 689 640006
792 4207 640006
4267 3714 640006
5062 3121 640006
5319 4404 640006
1056 1881 640006
8042 1508 640006
4771 1091 640006
2874 1393 640006
437 921 640006
5161 1353 640006
5062 3171 640006
1769 269 640006
3574 1674 640006
677 2712 640006
7050 467 640006
5862 2926 640006
4999 3888 640006
3436 6114 640006
5430 462 640006
3407 5826 640006
8854 325 640006
7553 1626 640006
2989 3297 640006
5887 2939 640006
5862 3993 640006
1031 3740 640006
802 5712 640006
4126 1520 640006
3980 2306 640006
8492 640 640006
3268 99 640006
2874 6305 640006
833 223 640006
8109 717 640006
5379 4476 640006
1056 7246 640006
4421 1461 640006
8771 55 640006
974 5661 640006
3268 3499 640006
7882 2107 640006
1056 5711 640006
5414 833 640006
2352 585 640006
833 4891 640006
3389 737 640006
3574 5127 640006
4267 3966 640006
3728 4416 640006
2352 4661 640006
3728 4154 640006
677 2759 640006
2819 2180 640006
4304 4550 640006
7250 2473 640006
437 4334 640006
2359 2121 640006
4480 4753 640006
3233 5593 640006
2505 5012 640006
2948 4302 640006
5655 2915 640006
5511 4344 640006
3389 3378 640006
51 2412 640006
9580 409 640006
1550 1282 640006
8183 309 640006
2937 3698 640006
2505 484 640006
2041 6660 640006
6722 45 640006
145 5579 640006
1480 818 640006
3761 3489 640006
5877 2911 640006
7517 525 640006
5248 2985 640006
2989 378 640006
4701 3618 640006
5655 3477 640006
260 4117 640006
5655 2979 640006
3574 847 640006
4421 5159 640006
1110 1838 640006
6247 2607 640006
3443 1258 640006
799 5715 640006
260 2572 640006
3153 6568 640006
2352 727 640006
833 5049 640006
3989 1761 640006
8319 535 640006
3443 861 640006
3574 2712 640006
2041 5209 640006
974 8747 640006
2463 914 640006
2352 2156 640006
8771 934 640006
799 1239 640006
260 7257 640006
6286 2064 640006
5379 4610 640006
4701 657 640006
2989 5782 640006
8183 1378 640006
3412 1967 640006
3153 3614 640006
1038 7663 640006
3436 831 640006
2948 3774 640006
2937 4313 640006
1550 3199 640006
2819 1127 640006
833 7276 640006
5319 1328 640006
148 8553 640006
1480 6753 640006
3989 5996 640006
4480 3753 640006
4604 5117 640006
3980 2267 640006
8144 490 640006
462 5262 640006
634 9087 640006
2756 6477 640006
7497 736 640006
51 2705 640006
4999 1287 640006
2951 1426 640006
8042 144 640006
1480 5570 640006
118 5759 640006
3377 1981 640006
5319 4386 640006
1550 1439 640006
1954 551 640006
1620 1816 640006
1480 5250 640006
3412 6309 640006
118 4259 640006
792 4932 640006
2756 2306 640006
1594 1980 640006
2819 6731 640006
2906 2508 640006
1439 1550 640006
5646 3241 640006
6635 1935 640006
4304 300 640006
799 175 640006
2298 4715 640006
3389 4930 640006
677 1090 640006
437 5373 640006
51 2900 640006
7050 1300 640006
2041 778 640006
4126 3597 640006
3377 5025 640006
5887 5 640006
8437 1517 640006
1031 8824 640006
9350 255 640006
2906 6674 640006
5877 1640 640006
3443 1161 640006
3367 709 640006
1594 8129 640006
5358 4222 640006
4604 4184 640006
1110 2297 640006
6635 1802 640006
8101 344 640006
799 2608 640006
8302 332 640006
2832 42 640006
1038 7750 640006
1594 6843 640006
2948 6773 640006
833 3647 640006
677 3831 640006
802 2146 640006
8652 698 640006
1031 8692 640006
6647 3342 640006
1038 1425 640006
7497 1204 640006
1480 1100 640006
2937 6768 640006
2298 521 640006
118 8973 640006
802 4517 640006
3761 616 640006
2756 1190 640006
3849 5283 640006
2906 5981 640006
2463 6325 640006
8101 600 640006
3989 5732 640006
1769 6375 640006
437 396 640006
1031 2237 640006
4604 3966 640006
3980 9 640006
7497 2457 640006
1550 913 640006
7497 56 640006
792 4587 640006
3367 10 640006
1550 491 640006
974 8749 640006
2906 4344 640006
3153 4364 640006
2951 4602 640006
7250 1402 640006
3980 4253 640006
974 465 640006
2038 3024 640006
3367 3268 640006
677 2476 640006
8854 696 640006
118 344 640006
4126 4193 640006
634 6001 640006
118 5692 640006
3761 5126 640006
4508 1003 640006
2874 5618 640006
2580 1496 640006
8302 1553 640006
2832 6718 640006
6647 603 640006
8437 389 640006
1620 4026 640006
2906 1602 640006
3574 2313 640006
3079 1401 640006
462 2357 640006
3980 4846 640006
8042 1538 640006
2352 2156 640006
1038 4686 640006
8101 469 640006
5862 2981 640006
1769 4517 640006
792 8558 640006
4076 1734 640006
2041 6278 640006
6514 1630 640006
8042 1679 640006
4999 1248 640006
634 2185 640006
51 8394 640006
3989 5734 640006
8634 946 640006
1336 6645 640006
3407 4574 640006
5161 2698 640006
3233 3402 640006
4604 4167 640006
2756 3974 640006
4421 1826 640006
6730 1152 640006
2359 720 640006
8319 507 640006
1620 2860 640006
4601 3543 640006
118 5774 640006
5358 3079 640006
5161 3025 640006
1038 2339 640006
6647 870 640006
2352 3934 640006
8843 390 640006
8101 301 640006
2937 2574 640006
634 5258 640006
3443 1306 640006
802 7850 640006
2298 6194 640006
1110 440 640006
2505 4217 640006
634 2755 640006
1056 2097 640006
3233 5946 640006
2756 118 640006
9091 259 640006
2505 4262 640006
3389 6332 640006
7859 1721 640006
2359 6495 640006
3443 6412 640006
3447 3200 640006
2505 3245 640006
799 9056 640006
2874 3003 640006
1439 7448 640006
1620 8334 640006
148 3288 640006
3079 1631 640006
4421 183 640006
2906 2605 640006
3447 5324 640006
3268 4229 640006
3377 6222 640006
799 7603 640006
462 977 640006
1056 7294 640006
8887 292 640006
2038 6663 640006
4126 5595 640006
5825 37 640006
792 2597 640006
3389 6191 640006
4480 3957 640006
3946 43 640006
833 7400 640006
2359 5158 640006
5430 447 640006
4076 2571 640006
802 534 640006
3079 6271 640006
7250 2704 640006
5319 4536 640006
3443 4 640006
1767 1052 640006
8570 273 640006
4421 87 640006
3761 1301 640006
8350 1635 640006
2505 1069 640006
1769 7363 640006
1594 1280 640006
2041 2439 640006
792 4527 640006
3436 553 640006
2906 1083 640006
3099 2220 640006
1038 8685 640006
1056 1763 640006
4710 1937 640006
6635 1802 640006
4421 2829 640006
51 2412 640006
118 1476 640006
3153 1355 640006
3447 1967 640006
3153 4706 640006
3099 2062 640006
3980 3902 640006
118 8184 640006
7517 833 640006
2298 5803 640006
1110 1969 640006
3079 6501 640006
1038 7399 640006
634 3094 640006
118 8201 640006
6247 2387 640006
148 4360 640006
1769 6883 640006
1439 4423 640006
2580 653 640006
3407 29 640006
462 1897 640006
2874 2637 640006
9605 384 640006
974 1782 640006
5414 3440 640006
51 4659 640006
792 4566 640006
3980 441 640006
260 5550 640006
2874 3018 640006
2463 2247 640006
974 620 640006
3367 5520 640006
1056 2311 640006
1954 1413 640006
3389 5013 640006
3407 1841 640006
2359 515 640006
4601 647 640006
2832 2993 640006
1031 8202 640006
1056 6497 640006
2937 1571 640006
1550 7084 640006
4480 4363 640006
2989 991 640006
7250 267 640006
5430 2429 640006
5511 2371 640006
2505 2743 640006
5887 2956 640006
6635 132 640006
2505 5048 640006
3574 4828 640006
5655 1842 640006
1110 928 640006
2359 4408 640006
1358 3890 640006
4701 1023 640006
4480 5100 640006
1594 4920 640006
1767 2982 640006
4710 3727 640006
8042 528 640006
5248 71 640006
802 1236 640006
2906 173 640006
4601 560 640006
1620 2757 640006
1031 2336 640006
4601 4100 640006
9561 428 640006
1620 1331 640006
2906 2919 640006
5877 2568 640006
3946 4404 640006
6730 3224 640006
6767 2324 640006
5724 2135 640006
3377 6577 640006
462 8771 640006
1620 6563 640006
1480 6839 640006
2906 5586 640006
677 8873 640006
4604 4746 640006
8302 143 640006
833 7049 640006
2874 4643 640006
3761 5126 640006
260 2559 640006
8402 1552 640006
2038 4609 640006
1031 3273 640006
3268 2090 640006
3412 6193 640006
3728 752 640006
6247 1795 640006
2906 1865 640006
6635 132 640006
634 1320 640006
3268 2162 640006
1550 1839 640006
3946 1936 640006
1550 3880 640006
1110 928 640006
2819 414 640006
2989 5448 640006
1767 5246 640006
1031 2336 640006
2948 6143 640006
5161 2562 640006
1031 5216 640006
148 6582 640006
5862 868 640006
1031 3346 640006
792 7700 640006
2874 2287 640006
833 7486 640006
2951 6140 640006
3980 4808 640006
4508 740 640006
1954 4768 640006
260 850 640006
5414 3023 640006
1110 7733 640006
2951 7034 640006
2874 5309 640006
3079 5030 640006
1358 6828 640006
4267 5083 640006
4999 826 640006
2038 5212 640006
1056 2890 640006
4126 1624 640006
8101 1504 640006
1954 7396 640006
4999 1768 640006
6722 3133 640006
1620 2684 640006
3443 2439 640006
3099 6886 640006
3389 1215 640006
8634 971 640006
802 2610 640006
2580 3302 640006
634 4428 640006
2038 3686 640006
8101 469 640006
3728 4717 640006
8701 849 640006
3574 415 640006
5887 360 640006
2505 2096 640006
1439 8141 640006
1358 5655 640006
1038 2195 640006
8634 599 640006
5655 1075 640006
51 4698 640006
792 2156 640006
2937 1330 640006
4701 3443 640006
1031 2381 640006
2874 7111 640006
437 4882 640006
5379 498 640006
2832 4891 640006
833 2603 640006
5892 2553 640006
5319 431 640006
2874 4623 640006
1550 6431 640006
8437 389 640006
2832 545 640006
148 5602 640006
2948 499 640006
5877 845 640006
799 6251 640006
1110 3494 640006
3153 4570 640006
1439 6443 640006
974 2754 640006
4999 3888 640006
2463 944 640006
3436 5914 640006
2832 5149 640006
792 6931 640006
3761 2121 640006
8788 773 640006
2298 1682 640006
5724 911 640006
4126 2160 640006
3946 6008 640006
2359 6275 640006
1550 8011 640006
2951 202 640006
5646 3915 640006
3980 1770 640006
51 8836 640006
4771 2726 640006
7250 1521 640006
2948 2934 640006
3946 2340 640006
2989 957 640006
3412 2099 640006
1550 8000 640006
7250 1451 640006
5414 3936 640006
1110 4401 640006
148 6366 640006
4421 183 640006
3079 2169 640006
2832 6347 640006
8826 1159 640006
1038 442 640006
2756 2405 640006
2038 3608 640006
118 3150 640006
2463 3183 640006
51 5307 640006
2041 6361 640006
8854 33 640006
8492 1497 640006
9605 384 640006
3574 4609 640006
2463 636 640006
974 7376 640006
5430 4555 640006
5892 2209 640006
462 7682 640006
2819 1127 640006
5161 485 640006
2352 3062 640006
1620 418 640006
145 4854 640006
3436 5655 640006
1336 3726 640006
2948 3819 640006
833 3544 640006
2298 3021 640006
4771 1106 640006
5882 2420 640006
8701 1022 640006
5646 1604 640006
462 371 640006
2463 980 640006
3728 5852 640006
2832 2993 640006
1769 272 640006
4267 1483 640006
3268 5302 640006
1767 187 640006
8437 389 640006
4604 5385 640006
4480 4374 640006
5430 4555 640006
1031 4130 640006
2463 7242 640006
5825 3755 640006
7250 247 640006
51 9299 640006
5750 132 640006
3233 4649 640006
2989 4992 640006
5414 4136 640006
8144 1461 640006
5655 859 640006
3099 5338 640006
6286 2947 640006
2038 3617 640006
2352 5881 640006
2874 562 640006
6514 2312 640006
5161 4389 640006
2874 1836 640006
1110 2158 640006
2906 3380 640006
2832 2993 640006
2832 2167 640006
118 2638 640006
3761 4541 640006
1336 8269 640006
802 2297 640006
118 4186 640006
1769 6212 640006
1031 6851 640006
974 4274 640006
2505 4142 640006
3079 6626 640006
4267 2247 640006
4304 3797 640006
1954 2172 640006
3574 1027 640006
2951 2297 640006
3268 1433 640006
8402 232 640006
5882 2555 640006
677 7509 640006
3268 6082 640006
5877 853 640006
7859 250 640006
260 7782 640006
1550 6887 640006
802 4853 640006
2832 6889 640006
1056 2212 640006
802 3144 640006
2937 5365 640006
3443 1915 640006
3079 1401 640006
1031 7670 640006
2937 5906 640006
6514 3047 640006
1358 4021 640006
1038 3038 640006
1056 2933 640006
3436 3286 640006
5877 370 640006
677 681 640006
2038 7141 640006
3443 2444 640006
7723 1857 640006
2989 3733 640006
7859 443 640006
1620 7613 640006
3761 719 640006
2832 2329 640006
7981 2008 640006
3099 977 640006
3443 6546 640006
5882 5 640006
51 1388 640006
4999 4990 640006
3443 537 640006
6635 2544 640006
2906 943 640006
437 9162 640006
3079 2169 640006
1550 2754 640006
802 2277 640006
8492 1107 640006
7981 807 640006
5892 1661 640006
7013 1306 640006
5511 4212 640006
7553 2008 640006
7497 689 640006
833 8890 640006
1620 8085 640006
3436 640 640006
8319 469 640006
1358 7044 640006
260 5251 640006
8319 1386 640006
1038 2405 640006
437 2930 640006
4701 3701 640006
462 569 640006
2038 3 640006
5379 498 640006
1336 3825 640006
8402 1583 640006
3079 288 640006
2580 5462 640006
3407 5030 640006
1439 8166 640006
3989 4053 640006
8101 391 640006
2951 4299 640006
1110 3370 640006
8826 265 640006
6767 1535 640006
1439 3919 640006
2906 2508 640006
2951 461 640006
802 2610 640006
1038 2811 640006
5414 478 640006
51 8299 640006
3099 2331 640006
5511 3141 640006
5511 314 640006
799 5078 640006
2298 3357 640006
1480 6243 640006
1031 408 640006
4508 3636 640006
2937 5172 640006
3980 4164 640006
2463 5518 640006
634 5253 640006
6730 1129 640006
8183 704 640006
2756 192 640006
677 4072 640006
7250 1983 640006
2038 3839 640006
4421 1456 640006
2989 6143 640006
1480 3124 640006
4076 528 640006
5430 3140 640006
833 6217 640006
1056 494 640006
3447 4070 640006
6730 1672 640006
5877 770 640006
1594 6550 640006
2874 4176 640006
1620 2888 640006
118 1836 640006
6730 1904 640006
1031 563 640006
3407 970 640006
3233 5337 640006
5724 1043 640006
1954 4693 640006
462 7035 640006
3268 3367 640006
1439 4386 640006
5319 4280 640006
2989 6610 640006
5430 3749 640006
833 3156 640006
2989 5837 640006
5379 267 640006
5655 3232 640006
1439 2541 640006
2937 6195 640006
3946 2568 640006
3377 5411 640006
3079 310 640006
5511 3722 640006
3389 2041 640006
1480 6870 640006
3412 3601 640006
4377 3809 640006
634 4745 640006
4377 1053 640006
2951 1798 640006
2359 7191 640006
2359 5683 640006
3436 5914 640006
1358 4467 640006
5414 4147 640006
2038 7512 640006
51 2855 640006
3233 5468 640006
3946 1916 640006
677 8149 640006
3412 1298 640006
51 2705 640006
5319 2198 640006
3849 4337 640006
8233 1490 640006
4421 1826 640006
9855 130 640006
1480 6243 640006
4377 394 640006
1031 3679 640006
3728 2154 640006
8570 609 640006
5430 2671 640006
3443 1328 640006
5430 817 640006
8771 809 640006
9132 589 640006
2906 1698 640006
3849 1581 640006
4601 3543 640006
4421 4350 640006
3153 1548 640006
3574 5252 640006
3728 4717 640006
2819 5063 640006
8109 1023 640006
7517 1117 640006
3436 6553 640006
1594 3116 640006
2874 1503 640006
4701 3601 640006
5161 649 640006
2463 1517 640006
8183 119 640006
1550 4736 640006
3989 1666 640006
2832 5570 640006
2463 5579 640006
3268 2243 640006
148 529 640006
1954 398 640006
3436 7 640006
1480 7221 640006
1439 1794 640006
1480 7407 640006
2505 314 640006
5062 815 640006
4377 3724 640006
3079 5808 640006
2756 5427 640006
8101 1031 640006
3574 4285 640006
3989 3024 640006
5825 2034 640006
2463 2238 640006
4126 3371 640006
4421 4711 640006
799 7503 640006
51 6679 640006
8183 704 640006
2463 5856 640006
634 1946 640006
6286 3275 640006
8843 289 640006
3153 5701 640006
1767 6275 640006
634 7408 640006
5248 639 640006
2832 247 640006
677 4753 640006
260 5632 640006
437 6613 640006
2937 1764 640006
7013 1173 640006
1769 6550 640006
5655 980 640006
145 3244 640006
2298 3527 640006
6286 2601 640006
3436 5335 640006
2948 6285 640006
8186 602 640006
3268 858 640006
974 5748 640006
2580 6599 640006
9561 38 640006
8233 1721 640006
9179 542 640006
1550 7276 640006
5750 2109 640006
2352 7633 640006
5062 1668 640006
1031 2543 640006
3407 6192 640006
5430 1205 640006
1038 5684 640006
799 1155 640006
