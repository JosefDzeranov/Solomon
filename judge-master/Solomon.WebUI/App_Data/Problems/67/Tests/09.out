23680222
37
330 698 137 43 705 812 298 284 519 816 317 472 945 433 31 312 98 74 517 879 420 784 268 79 176 66 713 315 754 937 539 142 675 77 395 989 910 